### PR TITLE
make statistic available to all drivers

### PR DESCRIPTION
**Pull request by sharonovd**

### CODE REVIEW

File: sharded_queue/drivers/fifottl.lua [hunk 1]
```diff
@@ -313,26 +252,21 @@ function method.put(args)
...
     for _, tuple in
-        box.space[args.tube_name].index.status:pairs(nil, {state.READY} ) do
-        if tuple ~= nil and tuple[index.status] == state.READY then
-            --
+        box.space[args.tube_name].index.status:pairs( {state.READY} ) do
+        if not is_expired(tuple) then
             task = tuple
             break
         end
     end
```
Comment (vanyarock01) [inline sharded_queue/drivers/fifottl.lua]: What if we could not find the ready task?

File: sharded_queue/storage.lua [hunk 2]
```diff
@@ -120,6 +104,35 @@ local function init(opts)
             rawset(_G, global_name, func)
             box.schema.func.create(global_name, { if_not_exists = true })
         end
+
+        local stats_func = function(args)
```
Comment (vanyarock01) [inline sharded_queue/storage.lua]: Move `stats_func` function to **statistic.lua**

File: test/ttl_test.lua [hunk 3]
```diff
@@ -9,6 +9,37 @@ g.before_all(function()
     g.queue_conn = config.cluster:server('queue-router').net_box
 end)
 
+local function lookup_task(task_id, tube_name, cluster)
+    local call_string = ("return box.space.%s:get(%s):tomap({names_only=true})"):format(tube_name, task_id)
```
Comment (olegrok) [inline test/ttl_test.lua]: nit: `['%s']` is a bit more safe than `.%s.`

Comment (sharonovd) [reply test/ttl_test.lua]: it's local helper in test :)

**Pull request merged**